client/src/components/About/About.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import CountUp from "react-countup";
+import ScrollTrigger from "react-scroll-trigger";
+
+function About() {
+  //to set state for numbers counting up on scrolling
+  const [counterOn, setCounterOn] = useState(false);
+
+  return (
+    <div>
+      {/* div for the main section */}
+      <div className="relative bg-[#354F52] h-screen">
+        <div className="font-['Raleway'] absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 text-4xl sm:text-6xl text-left text-[#84A98C]">
+          RENAISSANCE <br />
+          <span className=" text-white">7.0</span>
+        </div>
+      </div>
+      {/* div for the about section */}
+      <div className="md:flex md:flex-row md:items-center md:h-[30rem]">
+        {/* div for seat confirmation and heading */}
+        <div className="relative h-[18rem] md:w-1/3 md:h-[30rem]">
+          <div className="text-4xl font-['Raleway'] absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 text-left text-[#354F52] ">
+            ABOUT
+            <br />
+            DESIGN
+            <br />
+            WEEK
+            <br />
+            <button className="text-left text-[12px] text-white bg-[#354F52] p-2 hover:bg-[#2F3E46] md:w-[10rem]">
+              CONFIRM YOUR SEAT -{">"}
+            </button>
+          </div>
```

Comment:
Although it is as per figma, this doesn't look good. Try making the button a bit to the left.
![image](https://user-images.githubusercontent.com/77530596/217621683-cd400241-848b-4bb3-8945-ca351d1468ac.png)

Reply:
![Screenshot (219)](https://user-images.githubusercontent.com/93890098/218137019-a0492fb9-d829-4cde-b5f3-a3e5f8f87692.png)

Reply:
![Screenshot (220)](https://user-images.githubusercontent.com/93890098/218143470-929cd9f7-f6bb-4258-ac6c-980bbc517b68.png)